### DISCUSSION TITLE
[Feature] Consider switching to Weblate for translations

### DISCUSSION THREAD

**Discussion by nomorsug** [TechbeeAT/jtxBoard#919] [Ideas]
Weblate is free and opensource localization tool. The hosted version https://hosted.weblate.org/ is free for libre projects. It can also be selfhosted.

Compared to Crowdin it has larger pool of translators, it's easier to use and is used by the majority of FOSS android projects including F-Droid.

**Comment by patrickunterwegs** [comment]
Hi @nomorsug , thank you for your suggestion, actually I'm pretty happy with Crowdin, not sure if it's so much worth another change now...
I'll move this topic to discussion.

**Comment by keunes** [comment]
I'm supportive of this request mainly for principle reasons. But an added benefit of Weblate is that it's possible to add print screens to provide translators with context (which, as far as I'm aware, is not possible in CrowdIn).
